src/tachimawari/imu/node/imu_node.cpp

```diff
@@ -44,16 +44,16 @@ void ImuNode::update_imu()
 {
   auto unit_msg = kansei_interfaces::msg::Unit();
 
-  keisan::Vector<3> gyro = imu_provider->get_gyro();
-  keisan::Vector<3> accelero = imu_provider->get_accelero();
+  auto gyro = imu_provider->get_gyro();
```

Comment:
Sorry, maybe I'll change this and the related thing to not using `keisan` `Euler` and `Point3`. Because the value isn't degree, however, the value is a raw value from the sensor.